src/main/java/uk/nhs/hee/tis/trainee/reference/api/LocalOfficeContactResource.java

```diff
@@ -63,11 +64,24 @@ public LocalOfficeContactResource(LocalOfficeContactService service,
    * @return list of LocalOfficeContacts.
    */
   @GetMapping("/local-office-contact-by-lo-uuid/{localOfficeUuid}")
-  public List<LocalOfficeContactDto> getLocalOfficeContactsByLoUuid(
+  public List<LocalOfficeContactDetailsDto> getLocalOfficeContactsByLoUuid(
```

Comment:
This is the DTO that corresponds to the (enriched) LocalOfficeContact. The 'LocalOfficeContactDto' maps to the incoming data from the SQS queue, which is not enriched.